content/api/index.html

```diff
@@ -728,6 +728,13 @@
               specified using the thresholds option.
               <p>Example: <code>{'critical': 90, 'warning': 80}</code></p>
               </li>
+              <li><code>evaluation_delay</code> Time (in seconds) to delay
+              evaluation, as a non-negative integer.  For example, if the
+              value is set to 300 (5min) and the time is 7:00, the monitor
+              will evaluate data at 6:55. This is useful for AWS CloudWatch
```

Comment:
perhaps instead of 

> value is set to 300 (5min) and the time is 7:00

something like:

> value is set to 300 (5min), the timeframe is set to `last_5m` and the time is 7:00, the monitor will evaluate data from 6:50 to 6:55.